include/pokemon.h

```diff
@@ -467,7 +467,8 @@ struct MoveInfo
     u32 strikeCount:4; // Max 15 hits. Defaults to 1 if not set. May apply its effect on each hit.
     u32 criticalHitStage:2;
     u32 alwaysCriticalHit:1;
-    // 14 bits left to complete this word - continues into flags
+    u32 numAdditionalEffects:2; // limited to 3 - don't want to get too crazy
```

Comment:
From what I recall, the previous instance let us stack up to 15 secondary effects.
Can we preserve this, or would you prefer it to be done in a follow up PR?

Reply:
I would rather prefer if someone else then me does it (CF?)

Reply:
gotcha

Reply:
I wrote 15 in the move guide so that should have been 4 bits. It's not like there was no room for it.